src/components/Detail/Detail.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { VideoPlayer } from './VideoPlayer';
+import { RelatedVideoList } from './RelatedVideoList';
+import { BackButton } from './styles';
+
+export const Detail = ({ handleGoBack, videoId, onVideoChanged }) => {
+  return (
+    <>
+      <BackButton type="button" onClick={handleGoBack}>
```

Comment:
You can make use of the Link component for handling redirects instead of handling it with local state